www/src/js/views/snap-details.js

```diff
@@ -0,0 +1,231 @@
+var Backbone = require('backbone');
+var React = require('react');
+var ReactBackbone = require('react.backbone');
+var CONF = require('../config.js');
+
+var Installer = require('../components/installer.js');
+
+function SnapSize(props) {
+  var model = props.model;
```

Comment:
For the functional components.. it's not necessary, but a bit more explicit/nicer to destructure the props into arguments you actually use. So SnapSize({model}). After a bit of reading; I think it's safe to do <SnapSize {...model.attributes} /> and SnapSize({size}). Then the first two lines disappear.